overview/api/index.md

```diff
@@ -68,6 +73,7 @@ The services listed above are:
 - `docker-runner` --- [Service for Running Sync Actions](/extend/common-interface/actions/)
 - `import` --- [Storage Importer Service](/integrate/storage/api/importer/)
 - `syrup` --- [Service for Running Components](/extend/docker-runner/)
+- `oauth` --- [OAuth Manager Service](/extend/common-interface/oauth/)

```

Comment:
tady https://developers.keboola.com/extend/common-interface/oauth/ nejsou zadny zmeny? me se zda, ze jsme cosi upravovali v te strukture

Reply:
A na toto mrknem. Vyhadzal by som odtial to "2" a "v2" minimalne.

Reply:
Trochu som to upravil a pridal som tam zmienku o tom, ze v tom configu musi byt `"version": 3` aby to pouzilo novu OAuth Broker.

Este teda rozmyslam, ci tam nespomenut aj to, ze to zatial funguje len cez feature projektu??

Reply:
no zalezi jak moc to rychle to chces tady mergovat resp. jestli ma smysl to mergovat dokud to je jen pres ficuru projektu - normalne to nedelame